### ISSUE TITLE
Assassin's Creed Origins: blurry textures

### ISSUE THREAD

**Issue by Adibo88** [doitsujin/dxvk#2551]
Running the game with DXVK enabled causes textures to appear blurry, which can be seen on the screenshot. Two pics attached for comparison (1-with, 2-without dxvk). It started after the GPU driver was updated. Before version 20.12.1 it was fine. All driver versions afterwards cause this issue. I rolled back to the old driver, 20.9.1, and there were no issues, with or without dxvk. Then I switched to the newer driver, version 22.3.1, and the issue came back. With DXVK, textures become blurry, regardless of the DXVK version. So apparently the problem emerges if dxvk is used with latest AMD drivers.

[ACOrigins_d3d11.log](https://github.com/doitsujin/dxvk/files/8311553/ACOrigins_d3d11.log)
[ACOrigins_dxgi.log](https://github.com/doitsujin/dxvk/files/8311555/ACOrigins_dxgi.log)
![with dxvk](https://user-images.githubusercontent.com/101966890/159170243-68892757-4052-4ccd-8a30-8d3fec9ebee0.png)
![without dxvk](https://user-images.githubusercontent.com/101966890/159170248-9151ef79-991c-4770-a29b-19b923293f3b.png)

### Software information
Assassin's Creed Origins, latest patch, medium settings (although all settings were tried).

### System information
- GPU: Radeon RX480
- Driver: 22.3.1
- OS version: Windows 10 21H2
- DXVK version: 1.10 (issue persists regardless of the version)

**Comment by doitsujin** [comment]
Probably the same issue as https://github.com/doitsujin/dxvk/issues/2540.

TL;DR DXVK will no longer work correctly on Windows AMD drivers in situations where anisotropic filtering is used, and we can't really do anything about it.

**Comment by Adibo88** [comment]
> Probably the same issue as #2540.
> 
> TL;DR DXVK will no longer work correctly on Windows AMD drivers in situations where anisotropic filtering is used, and we can't really do anything about it.

Works perfectly fine on the same hardware and drivers but with different titles tho, e.g. Far Cry 3, Call of Duty WW2, NFS HP 2010.

**Comment by K0bin** [comment]
Different games do different things.

**Comment by Adibo88** [comment]
> Probably the same issue as #2540.
> 
> TL;DR DXVK will no longer work correctly on Windows AMD drivers in situations where anisotropic filtering is used, and we can't really do anything about it.

Clarification:
Actually, it seems that the issue is not texture related. Take a look at the screenshots. It's something wrong with rendering.
![ACOrigins_2022 03 21-18 39_1](https://user-images.githubusercontent.com/101966890/159274354-7ef78478-9551-49a5-bf5d-f992c4f714a0.png)
looks like the issue is not texture related. Take a look at the screenshots. The ga
![ACOrigins_2022 03 21-18 39](https://user-images.githubusercontent.com/101966890/159274324-3b5cf21b-60e2-4f32-be31-febaf7b6377c.png)
me looks like a retro pixel game.
![ACOrigins_2022 03 21-18 38](https://user-images.githubusercontent.com/101966890/159274279-c7c66062-4ee1-4263-b164-db51e9613bc0.png)

**Comment by doitsujin** [comment]
Not sure what to tell you, the game works fine on other drivers.

**Comment by Ryusennin** [comment]
It looks more like an internal resolution issue.

I don't have any such bug on Vega + driver 22.3.1 + latest DXVK build.
![Assassin's Creed Origins_2022 03 22-16 23](https://user-images.githubusercontent.com/9519544/159517967-57cc3b0d-449b-464c-9747-f8ce6817c8e4.png)

**Comment by doitsujin** [comment]
Vulkan features are not the issue here, Polaris supports everything we need. DXVK would just not work otherwise.

**Comment by Blisto91** [comment]
@Adibo88 Can you try again with the latest driver.

**Comment by Adibo88** [comment]
I no longer own that VGA, but I have tried using the R7 260X instead, latest legacy driver 22.6.1 and dxvk 1.10.3, on the same computer, OS and same graphics settings. Graphics seem to be ok without any visible glitches.

**Comment by pcmaniac2022** [comment]
Im having the same rendering issue with RX 580 on the newest driver and with dxvk 1.10.3
Does anyone fix this issue?
